app/koi-client/src/page/@backoffice@screen@[partyId]/component/StockScreen/index.tsx

```diff
@@ -25,9 +25,19 @@ export default function StockScreen({ party }: Props) {
 
   const startedTime = dayjs(stock?.startedTime).toDate();
   const isTransaction = stock?.isTransaction ?? false;
+  const [time, setTime] = useState('');
 
-  const { seconds, minutes } = getDateDistance(startedTime, new Date());
-  const time = `${prependZero(minutes, 2)}:${prependZero(seconds, 2)}`;
+  useEffect(() => {
+    const updateTimer = () => {
+      const { seconds, minutes } = getDateDistance(startedTime, new Date());
+      setTime(`${prependZero(minutes, 2)}:${prependZero(seconds, 2)}`);
+    };
+
+    updateTimer();
+    const intervalId = setInterval(updateTimer, 1000);
+
+    return () => clearInterval(intervalId);
+  }, [startedTime]);
```

Comment:
https://github.com/omizha/sdc-stock/pull/9/files#r1894018159
위 덧글을 참고하여 다음과 같이 변경할 수 있습니다.

```suggestion
  useEffect(() => {
    const updateTimer = () => {
      setTime(getTimeDistanceWithCurrent(startedTime));
    };
    const intervalId = setInterval(updateTimer, 1000);
    return () => clearInterval(intervalId);
  }, [startedTime]);
```

Reply:
오호 수정했습니다!